src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -387,6 +376,9 @@ public static Complex Sqrt(Complex value)
             
         }
 
+        // This is the largest x for which (Hypot(x,x) + x) will not overflow. It is used for branching inside Sqrt.
+        private static readonly double sqrtLimit = Double.MaxValue / (Math.Sqrt(2.0) + 1.0);
```

Comment:
Style: this should be be prefixed with `s_` according to our guidelines. Could you actually rename this to something like `s_sqrtScaleThreshold` so it's more descriptive?